x-pack/plugins/security_solution/server/lib/detection_engine/rules/utils.ts

```diff
@@ -102,6 +102,13 @@ export const transformFromAlertThrottle = (
   legacyRuleActions: LegacyRuleActions | null | undefined
 ): string => {
   if (legacyRuleActions == null || (rule.actions != null && rule.actions.length > 0)) {
+    if (
+      rule.params?.type === 'query' &&
+      rule.params?.responseActions &&
+      rule.params?.responseActions.length > 0
+    ) {
+      return NOTIFICATION_THROTTLE_RULE;
+    }
```

Comment:
Why do we always return `NOTIFICATION_THROTTLE_RULE` if response actions exist? If a user sets regular actions with a throttle, won't this report an incorrect value when they read the rules back through the API?

Reply:
Good catch, I have to figure out a way to  avoid this then 
` if (rule.muteAll || rule.actions.length === 0) {
      return NOTIFICATION_THROTTLE_NO_ACTIONS;
    }`  

Will get back to you with a suggestion, thanks!

Reply:
https://github.com/elastic/kibana/pull/133279#discussion_r972903426